wiki/Article_Style_Guide/en.md

```diff
@@ -68,6 +79,22 @@ The title of an article is the name of the folder that the article is located in
 For English article titles, if you need to reword the title, you must rename the folder to match the article title.
 
 All article titles are to be using the level 1 heading.
+For example:
+
+```
+# Folder Name
+```
+
+In the case where a title cannot be translated, use the language title (in English) followed by the language name (in that language) in round brackets.
+For example:
+
+```
+# osu!mania (Español)
+# Live Streaming (日本語)
+# Skinning osu!catch (Français)
+```
+
+You can find a list of language names in their language at [Language names in their own languages and scripts](http://www.omniglot.com/language/names.htm) or [List of language names](https://en.wikipedia.org/wiki/List_of_language_names).
```

Comment:
Rewording to use `list of native languages`

---

~~but wouldn't it be more confusing if it had said `You can find a list of language names at `... (I mean, in which language... in English?)~~